Styles/docs.css

```diff
@@ -0,0 +1,130 @@
+.header {
+  z-index: 1;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 100%;
+  border-bottom: 1px solid var(--primary);
+  background-color: var(--secondary);
+}
+
+.brand-name {
+  font-family: "Audiowide", cursive;
+  margin: 1rem;
+  text-shadow: -3px 1px #00e1db;
+}
+
+.header-nav {
+  margin: auto 0;
+}
+
+.header-nav ul {
+  display: flex;
+}
+
+.active{
+  background-color: var(--primary);
+  color: var(--secondary);
+  border-radius: 10px;
+  font-weight: bold;
+}
+
+.wrapper{
+  display: grid;
+  grid-template-columns: 1fr 5fr;
+  margin-top: 6rem;
+  align-items: baseline;
+}
+
+aside li{
+  margin: 20px;
+}
+
+.main-container section {
+  display: block;
+  margin: 2rem;
+}
+
+h1{
+  color: var(--primary);
+}
+p {
+  margin-top: 1rem;
+}
+
+span {
+  font-size: 1rem;
+  color: var(--primary);
+}
+
+.demo-container {
+  display: flex;
+  align-items: flex-end;
+  justify-content: center;
+}
+
+.demo-container > * {
+  margin: 15px 5px;
+}
+
+footer{
+  text-align: center;
+}
+
+/* making image responsive */
+.img-responsive {
+  max-width: 100%;
+  height: auto;
+}
+
+/* highlight.js */
+pre {
+  white-space: pre-line;
+  position: relative;
+  max-width: 80%;
+  max-height: 350px;
+  overflow-y: auto;
+  margin: 1rem auto;
+}
+code {
+  line-height: 2;
+  padding: 2rem !important;
+}
+.snippet-btn {
+  background-color: transparent;
+  position: absolute;
+  cursor: pointer;
+  top: 7px;
+  right: 10px;
+}
+.snippet-btn > i {
+  font-size: 1.5rem;
+  color: white;
+}
+
+.snippet-btn:hover {
+  transform: scale(1.2);
+}
+
+#copiedSnackbar {
+  display: none;
+  color: black;
+  border: 2px solid black;
+  background-color: white;
+  border-radius: 2px;
+  padding: 10px;
+  position: fixed;
+  z-index: 10;
+  left: 50%;
+  top: 0;
+  font-size: 20px;
+}
+
+#copiedSnackbar.show {
+  display: block;
+  -webkit-animation: fadein 0.5s, fadeout 0.5s 2.5s;
+  animation: fadein 0.5s, fadeout 0.5s 2.5s;
+}
```

Comment:
Do not use ID. More on this here https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4

Reply:
This was provided in the highlight.js guide for the code block. So, I haven't changed much here.